docs/docker-images/all-in-one.md

```diff
@@ -0,0 +1,87 @@
+
+#### -t
+
+This is the easiest way to ensure you get the nice, colorized output from `earthly`. Alternatively, you could provide the `FORCE_COLOR` environment variable.
+
+### Supported Environment Variables
+
+| Variable Name                       | Default Values                 | Description                                                                                                                                                                                           |
+|-------------------------------------|--------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| GLOBAL_CONFIG                       |                                | Any valid YAML for the top-level `global` key in `config.yml`. Example: `{disable_analytics: true, local_registry_host: 'tcp://127.0.0.1:8371'}`                                                      |
+| GIT_CONFIG                          |                                | Any valid YAML for the top-level `git` key in `config.yml`. Example: `{example: {pattern: 'example.com/([^/]+)', substitute: 'ssh://git@example.com:2222/var/git/repos/\$1.git', auth: ssh}}`         |
+| NO_DOCKER                           |                                | Disables the check for a working Docker Daemon. Setting this _at all_ disables the check.                                                                                                             |
+| DOCKER_HOST                         | `/var/run/docker.sock`         | From Docker's CLI.                                                                                                                                                                                    |
+| BUILDKIT_HOST                       | `tcp://<hostname>:8372`        | The address of your buildkit host. Use this when you have a remote `buildkitd` you would like to connect to.                                                                                          |             
+| SRC_DIR                             | `/workspace`                   | The working directory for `earthly`. Usually host-mounted.                                                                                                                                            |
+| EARTHLY_ADDITIONAL_BUILDKIT_CONFIG  |                                | Additional `buildkitd` config to append to the generated configuration file.                                                                                                                          |
+| BUILDKIT_LOCAL_REGISTRY_LISTEN_PORT | `8371`                         | What port should the internal cache registry listen on?                                                                                                                                               |
```

Comment:
I think this should be disabled for both images. The reason is that Earthly code assumes the registry is at `127.0.0.1:8371` and doesn't support anything else. If it's enabled, buildkit will try to export via registry and earthly would not be able to receive the images, leading to failures.

Reply:
I haven't touched this setting at all; this was just documenting what was there. You want me to actively force it to the default value of 8371?

Reply:
Perhaps don't document its existence just yet. The default should be unset, in which case buildkit will not use a registry for exports.

Reply:
It remains unset in here. The defaults in this table arent always from my script; but rather are what they end up defaulting to if unspecified... eventually. I will remove this from the config.